lib/bcdice/game_system/DungeonsAndDragons5.rb

```diff
@@ -191,6 +197,46 @@ def ability_roll(command)
           end
         end
       end
+
+      # 武器の両手持ちダメージ
+      def twohands_damage_roll(command)
+        m = /^2H(\d+)D(\d+)([+-]\d+)?/.match(command)
+        unless m
+          return nil
+        end
+
+        dice_count = m[1].to_i
+        dice_number = m[2].to_i
+        modify = m[3].to_i
+        mod_str = number_with_sign_from_int(modify)
+        output = ["(2H#{dice_count}D#{dice_number}#{mod_str})"]
+
+        dice = @randomizer.roll_barabara(dice_count, dice_number)
+        roll_dice = "[" + dice.join(",") + "]"
+        output.push("#{roll_dice}#{mod_str}")
+
+        ex_dice = []
+        new_dice = []
+        sum_dice = 0
+        dice.each do |num|
+          if num.to_i > 2
+            sum_dice += num.to_i
+            ex_dice.push(num)
+          else
+            one_die = @randomizer.roll_once(dice_number)
+            sum_dice += one_die.to_i
+            new_dice.push(one_die)
+          end
+        end
+        if new_dice.!empty?
```

Comment:
```suggestion
        if new_dice.empty?
```
記述ミスかと思いますので修正お願いします

Reply:
if じゃなくて unless を使った方が良いということでしょうか？
実は素では「if new_dice.length > 0」と書いていたらrubocop先生に「そこは .!empty? だろ」と訂正されたのでしたｗ

Reply:
いえ、単に文法上 `.!empty?` という記法がないです。そのためテストも全て失敗しています。 

`if !new_dice.empty?`が正しかったんですね。 `unless` と `if !` のどちらでも問題ないです、

Reply:
なんと！私の環境ではテストを通ってました。(流石の私も、テストしないでPull Requestは送れないですからね)
文法上存在しないものがrubocopから提示されるとは考えてもみなかったですね。
そういうこともあるんだなぁ。